products/yocto.md

```diff
@@ -19,6 +19,13 @@ auto:
 
 # for eol see https://wiki.yoctoproject.org/wiki/Releases
 releases:
+-   releaseCycle: "4.3"
+    codename: 'Nanbield'
+    releaseDate: 2023-11-09
+    eol: 2024-04-09
```

Comment:
I am unsure about this date. Have you a source for that?

https://wiki.yoctoproject.org/wiki/Releases states:
> Support for 7 months (until May 2024)

But releaseDate(4.3) plus 7 months is [2024-06-09](https://www.timeanddate.com/date/dateadded.html?d1=9&m1=11&y1=2023&am=7).

Reply:
Uhm. Yes, you're right.

Reply:
The release was scheduled for October, but it was pushed into November somehow. Nevertheless, the +7 months rule still holds.